src/jit/lclvars.cpp

```diff
@@ -3670,13 +3621,21 @@ var_types LclVarDsc::lvaArgType()
     return type;
 }
 
-/*****************************************************************************
- *
- *  This is called by lvaMarkLclRefsCallback() to do variable ref marking
- */
+//------------------------------------------------------------------------
+// lvaMarkLclRefs: increment local var references counts and more
+//
+// Arguments:
+//     tree - some node in a tree
+//     block - block that the tree node belongs to
+//     stmt - stmt that the tree node belongs to
+//
+// Notes:
+//     Invoked via the MarkLocalVarsVisitor
```

Comment:
Might be nice to add a bit more detail here about what the "and more" is - e.g. "In addition to increment local var reference counts, it identifies additional local var properties such as those used by assertion prop."

Reply:
Thanks. I was surprised how much it did and never got back to filling it all in. 

Ok if I do this in my next change? I need to make sure that the only required part for minopts is having nonzero ref counts.

Reply:
Fine with me, this is already way better than what was there!